Image_process/processImage.py

```diff
@@ -0,0 +1,136 @@
+import os
+
+from PIL import Image
+
+
+class ImageProcess:
+    def __init__(self, image_path):
+        self.image_path = image_path
+
+    def resize_image(self, width=128, height=128):
+        """
+        Resize the image to the specified width and height.
+        :param width: Target width of the image.
+        :param height: Target height of the image.
+        :return: Resized Image object.
+        """
+        if width <= 0 or height <= 0:
+            raise ValueError("Width and height must be positive integers.")
+        #if image isn't square raise a warning and resize to square
+        image = Image.open(self.image_path)
+        if image.size[0] != image.size[1]:
+            print("Warning: Image is not square. Resizing to square.")
+
+        return image.resize((width, height))
+    
+    def save_image(self, image, save_path):
```

Comment:
As per my last comment, this is misplaced responsibility. The image transformer should no nothing about the underlying medium images get saved to, it might save them to memory instead of disk or to object storage.